tests/providers/snowflake/hooks/test_snowflake.py

```diff
@@ -121,6 +121,7 @@ def test_get_conn_params(self):
             'role': 'af_role',
             'authenticator': 'snowflake',
             'session_parameters': {"QUERY_TAG": "This is a test hook"},
+            "application": os.environ.get("AIRFLOW_SNOWFLAKE_PARTNER", "AIRFLOW"),
```

Comment:
IMO it’d be better to mock the environ and also test the logic is correct.

Reply:
Let me do it @ephraimbuddy . My fault.

Reply:
@uranusjr I'm not sure I understand the logic you want tested. I have just mocked it